src/Todo/Model.hs

```diff
@@ -0,0 +1,25 @@
+module Todo.Model where
```

Comment:
Could you please also specify explicit export list? This is a good design for projects. And it helps in future. To export data type and all its fields and constructors you need to write something like that:

```haskell
module Todo.Model where
       ( Todo (..)
       , State (..)
       , Title (..)
       -- the rest
       ) where
```